Makefile

```diff
@@ -0,0 +1,8 @@
+TEST_ARGS ?= ""
```

Comment:
You can also have a look at the tox.ini file https://github.com/flightctl/flightctl-ansible/blob/main/tox.ini